gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy

```diff
@@ -346,6 +346,8 @@ subprocess.check_call([
     }
 
     void apply(Project project) {
+        project.ext.glean_version = "20.0.0-SNAPSHOT"
```

Comment:
Yes, I'm not too much against this either. Looks like `android-components` is able to read the content of the `.buildconfig.yml` from [gradle](https://searchfox.org/mozilla-mobile/source/android-components/buildSrc/src/main/java/Config.kt#8). We could probably [automate that](https://searchfox.org/mozilla-mobile/rev/392e4629e2dcf31cd4d5e685d54fc595029c08ad/android-components/settings.gradle#32-44) without resorting to scripts?

Reply:
Yeah -- but in this case we need to get a value from the gradle configuration (doesn't really matter where it lives) into the code of the plugin.  That's what the "buildconfig" plugins all try to solve.  But it doesn't work because our plugin is used both as a script (internally from glean) and as a proper plugin-in-a-jar from third parties.

Gradle always has these bizarre limits on what works where... ;)

Reply:
Ok, I'm fine with landing this as it is and try to "improve later" :)

Reply:
Let's add this as a requirement in https://mozilla.github.io/glean/book/dev/cut-a-new-release.html